features/zones/DetailViewContent.feature

```diff
@@ -4,38 +4,6 @@ Feature: Zones: Detail view content
       | Alias   | Selector                            |
       | details | [data-testid='detail-view-details'] |
 
-  Scenario Outline: Zone CP detail view has expected content
-    Given the URL "/config" responds with
-      """
-      body:
-        mode: global
-      """
-    And the URL "/zones+insights/zone-cp-1" responds with
-      """
-      body:
-        name: zone-cp-1
-        zoneInsight:
-          subscriptions:
-            - connectTime: 2020-07-28T16:18:09.743141Z
-              disconnectTime: 2020-07-28T16:18:09.743141Z
-              status: {}
-            - connectTime: 2020-07-28T16:18:09.743141Z
-              status: {}
-      """
-
-    When I visit the "/zones/zone-cps/zone-cp-1" URL
-    Then the page title contains "Zone CP"
-    Then the "$details" element contains "Zone CP: zone-cp-1"
-    Then the "$details" element contains "ZoneOverview"
-    Then the "$details" element contains "online"
-    Then the "$details" element contains "dpToken"
-
-    When I click the "#insights-tab" element
-    Then the "$details" element contains "Connect time: July 28, 2020 at 4:18:09 PM"
-
-    When I click the "#config-tab" element
-    Then the "$details" element contains "adminAccessLogPath"
-
```

Comment:
I moved this into `features/zones/zone-cps/Item.feature` while I was here (we should also move the others to `features/zones/zone-ingresses` and `features/zones/zone-egresses` at some point)